test/e2e/mock-e2e.js

```diff
@@ -20,6 +20,11 @@ const {
   DEFAULT_FEATURE_FLAGS_RESPONSE: BRIDGE_DEFAULT_FEATURE_FLAGS_RESPONSE,
 } = require('./tests/bridge/constants');
 
+const {
+  WHITE_LISTED_HOSTS,
+  WHITE_LISTED_URLS,
+} = require('./mock-e2e-whitelisted');
```

Comment:
Nit: We should prefer using more inclusive language, "allowlist" over "whitelist". I see "backlist" is already used though, so maybe this was meant to follow that pre-existing convention. But we already use blocklist/allowlist in most places, e.g. the phishing config.

Reply:
sure make sense! Changing it now :pray: